src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -39855,6 +39855,94 @@ static void Main()
             }
         }
 
+        [Fact]
```

Comment:
> [Fact] [](start = 7, length = 7)

Not related to this PR: have we thought about the use of Obsolete attribute in DIM? Consider adding some tests if interesting.

Reply:
> Not related to this PR: have we thought about the use of Obsolete attribute in DIM? Consider adding some tests if interesting.

Unless you have something specific in mind, I do not believe there is specific interaction between DIM and Obsolete attribute. 

---
In reply to: [288299204](https://github.com/dotnet/roslyn/pull/35957#discussion_r288299204) [](ancestors = 288299204)